### DISCUSSION TITLE
possible to use keepassxc for browser popup logins?

### DISCUSSION THREAD

**Discussion by prankousky** [keepassxreboot/keepassxc#9067] [Q&A] [answered]
Hi everybody,

is it possible to configure `keepassxc` so that it will automatically fill browser popup login forms? Here is an example

![grafik](https://user-images.githubusercontent.com/11297070/216815340-ca225054-2bc8-48b3-9202-72c6ee6b7eeb.png)

This popup will appear as soon as I type in the URL; it is not an html document, but wants me to authenticate **before** displaying any html. In this case, it is a login form for a `tasmota` device, but other services, for example, `sonarr` have this kind of authentication as well...

I have the URL set, but there is no way to automatically fill these kinds of form - or hopefully I just missed it, and it is actually possible?

Thank you in advance for your help :)

**Comment by droidmonkey** [comment]
Yes this is possible, you just need to enable the option in the browser settings:


![image](https://user-images.githubusercontent.com/2809491/217384102-86abd682-1630-4d27-87a9-49e426505e3d.png)

**Reply by prankousky** [reply]
Thank you. I checked. This function **had** been enabled already, but these popups would not be filled in. I use `KeePassXC-Browser` as firefox plugin for this.

**Comment by millhaus** [comment, marked as answer]
You have to adjust this in the settings of the browser plugin, not in KeepassXC itself.
For Firefox see attached screenshot (sorry I only have a german screenshot), there is a checkbox something like `allow fill http basic auth data` or similar.

![KeepassXC_enable_http_login_popup](https://github.com/user-attachments/assets/55c54624-ceae-4fe2-aa93-1b2e94aa328e)

**Reply by prankousky** [reply]
Thank you. You are right, my box was unchecked, it should work fine now.

Und meine KeePass Einstellungen sehen Deinen sowieso ähnlicher, als der englischsprachigen Variante ^^
